AustrianPatient/patientReligion.StructureDefinition.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <url value="http://hl7.at/fhir/3.0/StructureDefinition/patientReligion" />
+  <name value="patientReligion" />
+  <status value="draft" />
+  <fhirVersion value="3.0.1" />
+  <mapping>
+    <identity value="rim" />
+    <uri value="http://hl7.org/v3" />
+    <name value="RIM Mapping" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <contextType value="resource" />
+  <context value="Patient" />
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension.extension">
+      <path value="Extension.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Extension.extension:code">
+      <path value="Extension.extension" />
+      <sliceName value="code" />
+    </element>
+    <element id="Extension.extension:code.url">
+      <path value="Extension.extension.url" />
+      <fixedUri value="code" />
+    </element>
+    <element id="Extension.extension:code.value[x]:valueCodeableConcept">
+      <path value="Extension.extension.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <description value="https://termpub.gesundheit.gv.at/TermBrowser/gui/main/main.zul?loadType=ValueSet&amp;loadName=ELGA_ReligiousAffiliation" />
+        <valueSetUri value="urn:oid:2.16.840.1.113883.2.16.1.4.1" />

```

Comment:
This is the URI for the CodeSystem.
The correct URI for the ValueSet would be:
```suggestion
        <valueSetUri value="urn:oid:1.2.40.0.34.10.18" />
```

Reply:
@spodskubka thanks for that. However, as discussed in one of the previous telcos, we rely on the HL7 ReligionAustria Code System for the AustrianPatient, rather than the ELGA-ValueSet. Therefore, you are completely right, things are not correct, but it is the URL that needs to be changed accordingly: https://termpub.gesundheit.gv.at:443/TermBrowser/gui/main/main.zul?loadType=CodeSystem&loadName=HL7.AT:ReligionAustria

Reply:
@aschuler84 it seems I mixed up the correct solution. Thanks for fixing it correctly!